rfd/0043-kubeaccess-multiparty.md

```diff
@@ -144,18 +144,38 @@ If tap is made after the alert, the follwing message is shown:
 Teleport >> MFA tap received.
 ```
 
+#### Broadcast messages
+
+For TTY-enabled SSH and Kubernetes sessions that require additional participants as per a require policy, Teleport will now inject broadcast messages into the session to notify participants
+about the state of the session. No messages will be shown in sessions that do not require additional participants.
+
+This is needed so that participants are aware of what is currently happening in the session
+for security and usability reasons.
+
+Each broadcast message is prefixed with `Teleport > ` to indicate that this message
+is injected by Teleport and not from the originating shell.
+
+There are 9 kinds of broadcast messages:
+- `Creating session with uuid <example-uuid>...`: Sent on session creation.
+- `User <user> joined the session.`: Sent when a user joins the session.
+- `User <user> left the session.`: Sent when a user leaves the session.
+- `Lanching session...`: Sent when the session is launched and thus tranferred to a normal shell.
+- `Session closed.`: Sent when the session is closed due to the termination of the shell.
+- `Session forcefully terminated`: Sent when moderator forcefully terminates a session.
+- `Session paused, waiting for additional participants...`: Sent when a session is paused due to lack of required participants.
+- `Session resumed.`: Sent when a session has the required participants and is resumed.
+- `Please tap your MFA key within 15 seconds.` and `Teleport >> MFA tap received.` are messages shown to moderators in sessions which are MFA-presence enabled.
+
 ##### Example
 
-This example illustrates how a group 3 users of which Alice is the initiator and Eve and Ben are two observers
-start a multiparty session. Below is a a series of events that happen that include what each user sees and what
-they do.
+This example illustrates how a group 3 users of which Alice is the initiator and Eve and Ben are two observers start a multiparty session. Below is a a series of events that happen that include what each user sees and actions taken.
```

Comment:
Are _observers_ always moderators? I suggest being more clear here. For example:

```suggestion
This example illustrates how a TTY-enabled SSH that requires a moderator works. In this example, Alice is a participant and both Ben and Eve are moderators. Below is a series of events that happen and what each user sees as a result.
```